src/Data/Argonaut/Decode/Class.purs

```diff
@@ -12,7 +12,8 @@ import Data.Int (fromNumber)
 import Data.List (List(..), (:), fromFoldable)
 import Data.Map as M
 import Data.Maybe (maybe, Maybe(..))
-import Data.String (charAt)
+import Data.String (CodePoint)
+import Data.String.CodePoints (codePointAt)
```

Comment:
Is this not exported by `Data.String`?

Reply:
yes that's possible - I recheck and change (my tooling got broken, so I don't see the warnings right now)

Reply:
Looks to me like `Data.String` exports everything from `Data.String.CodePoints`: https://github.com/purescript/purescript-strings/blob/80dcf7ef362477214bee9082de091331576feb76/src/Data/String.purs

Reply:
I just changed this one minute ago :D